game.h

```diff
@@ -22,7 +22,7 @@ class game
   game(double wall_short_side = 1600,
        int num_players = 3,
        int n_ticks = 0,
-       std::size_t n_shelters = 3,
+       std::size_t n_shelters = 42,
```

Comment:
Instead of setting a number of shelters across all tests manually, why don't we create a global variable(is this bad?) at the beginning of game.h file?

Reply:
First, we should [I.2: Avoid non-const global variables](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Ri-global). Second, there is now exactly one place where the default number of shelters is set, as a default argument, at the expected function (i.e. the constructor). I would say this is as good as it gets (although I know 1 equivalently good syntax)